sycl/doc/extensions/experimental/sycl_ext_oneapi_virtual_mem.asciidoc

```diff
@@ -380,7 +380,7 @@ the range is mapped to.
 |`void unmap(const void *ptr, size_t numBytes, const device &syclDevice, const context &syclContext)` |
 Unmaps the range specified by `ptr` and `numBytes`. The range must have been
 mapped through a call to `physical_mem::map()` prior to calling this. The range
-must not be a proper sub-range of a previously mapped range. `syclContext` must
+must be a proper sub-range of a previously mapped range. `syclContext` must
```

Comment:
I don't think this is right. The intention of the sentence is to require that the range is the full range previously mapped, i.e. the full range is not a proper sub-range.

Reply:
Yeah, I also wasn't sure if that's a correct change. However:

> The intention of the sentence is to require that the range is the full range previously mapped

Why can't we just say that directly without negation? The phrase that it must not be a proper sub-range reads to me as: "it can be an invalid sub-range" which is not something we intended it to be, I guess

Reply:
I can see how it could be hard to read. What rephrasing do you propose?

Reply:
seems like 383 was a copy paste from line 220 https://github.com/intel/llvm/pull/15505/files#diff-f4e4beec40c0493fae3c83bc12ce8cec99d383aa554d3874715716f18992ffdbR220